changelogs/unreleased/9600-use-the-product-module-constraints.yml

```diff
@@ -0,0 +1,8 @@
+description: >
+  Modules constraints defined at the product level in the compatibility.json file are now
+  enforced during project and agent install.
```

Comment:
I suppose there is already a change entry about the introduction of the file? Perhaps we should just update that one?

Reply:
I agree :+1:

Reply:
Arf, the other one is master only

Reply:
Ah, I see. I agree then. You'll have to cherry-pick most of the other PR to iso8 then.

Reply:
Cherry pick [PR](https://github.com/inmanta/inmanta-core/pull/9620)